src/components/members/MemberScreen.js

```diff
@@ -52,23 +57,72 @@ const MemberScreen = () => {
           m={1}
           bgcolor="background.paper"
         >
-          <AccountCircleIcon className={classes.icon} size />
-          <Typography>{`Id: ${member?.id}`}</Typography>
-          <Typography>{`Name: ${member?.name}`}</Typography>
-          <Typography>{`Email: ${member?.email}`}</Typography>
-          <Typography>{`Type: ${member?.type}`}</Typography>
+          <AccountCircleIcon className={classes.icon} />
+          <Typography>{`Id: ${member.get('id')}`}</Typography>
+          <Typography>{`Name: ${member.get('name')}`}</Typography>
+          <Typography>{`Email: ${member.get('email')}`}</Typography>
+          <Typography>{`Type: ${member.get('type')}`}</Typography>
           <Typography>
-            {`Created At: ${formatDate(member?.createdAt)}`}
+            {`Created At: ${formatDate(member.get('createdAt'))}`}
           </Typography>
         </Box>
+        <Box
+          display="flex"
+          flexDirection="column"
+          p={1}
+          m={1}
+          bgcolor="background.paper"
+        >
+          <Typography>Extra:</Typography>
+
+          <ReactJson src={member.get('extra')} />
+        </Box>
       </Box>
-      <ItemsTable
-        id={buildItemsTableTitle(memberId)}
-        empty={false}
-        items={List(items)}
-      />
+      {!currentRoles.isEmpty() && (
+        <Box
+          display="flex"
+          flexDirection="row"
+          p={1}
+          m={1}
+          bgcolor="background.paper"
+        >
+          <List
+            component="nav"
+            aria-labelledby="nested-list-subheader"
+            subheader={
+              // eslint-disable-next-line react/jsx-wrap-multilines
+              <ListSubheader component="div" id="nested-list-subheader">
+                Roles
+              </ListSubheader>
+            }
+            className={classes.root}
+          >
+            {currentRoles.map((role, index) => {
+              const permissionsDescription = rolesPermissions
+                .get(index)
+                .map((p) => p.description);
+              return (
+                <ExpandableListItem
+                  key={role.description}
+                  itemName={role.description}
+                  content={permissionsDescription}
+                />
+              );
+            })}
+          </List>
+          <PermissionsTable
+            empty={false}
+            tableTitle={"Admin's Permissions"}
+            permissions={ListM(rolesPermissions.toArray().flat())}
```

Comment:
`ListM`?

Reply:
I would keep immutable's list as it is, and change material's as `ListComponent`

Reply:
What about immutable's `flatMap`/`flatten`?